include/Stand_der_Technik.tex

```diff
@@ -64,5 +65,7 @@ \section{Technische Konzepte}
 \end{table}
 
 \section{Stand im Bibliothekswesen}
-%TODO Pascal
+Im Bibliothekswesen ist RFID immer häufiger anzutreffen. Der Hauptvorteil liegt hier darin, dass Personalkosten eingespart werden können. Diese Einsparungen wird erreicht, durch mehrere nun automatisierte Prozesse, wie die selbst ausleihe eines Kunden \parencite{Keller2010}.
+Im Bibliothekswesen wird bereits seit den späten 1990 Jahren auf RFID zurückgegriffen. Zu diesem Zeitpunkt war es technisch jedoch nur möglich RFID HF zur verwenden. Daher wurde dies der De facto Standard im Bibliotheksumfeld \parencite{rfidHfVsUhf}. So ist es nicht verwunderlich, dass die meisten Komplett-System Anbieter in der Schweiz nur RFID HF als Lösung anbieten. Wie in Kapitel \ref{sec:technischeKonzepte} beschrieben beträgt die maximale erreichbare Distanz von RFID HF knapp einen Meter. Diesem Umstand ist wahrscheinlich zu verdanken, dass keine der gefunden Komplett-Systemanbieter, Identifikation von HF Tags in einem Hochregallager anbietet.
```

Comment:
Zu diesem Zeitpunkt war es aus technischen Gründen nur möglich HF RFID zu verwenden.

Reply:
Daher wurde HF RFID der **de facto** Standard im Bibliotheksumfeld.

Reply:
Generell HF RFID liest sich meiner Meinung nach besser. RFID HF hört sich na einer Produktbeschreibung in einem Warenkatalog an.

Reply:
**Dieser Einschränkung** ist wahrscheinlich **der Grund**, dass keine der gefundenen Komplett-Systemanbieter Identifikation von HF Tags in einem Hochregallager anbieten.